### PR TITLE
KAZOO-4170: Filter feature codes when checking if number already exis…

### PR DESCRIPTION
**Pull request by macpie**
…t when adding/updating a  callflow

### CODE REVIEW

File: core/kazoo_documents-1.0.0/src/kzd_callflow.erl [hunk 1]
```diff
@@ -17,11 +18,39 @@
...
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec is_feature_code(wh_json:object()) -> boolean().
+is_feature_code(JObj) ->
+    case wh_json:get_value(?FEATURE_CODE, JObj) of
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_callflow.erl]: `wh_json:get_is_true(?FEATURE_CODE, JObj, 'true')`

File: applications/crossbar/src/modules/cb_callflows.erl [hunk 2]
```diff
@@ -442,12 +443,14 @@ check_uniqueness(Numbers, JObjs, Context) ->
...
 check_numbers_uniqueness([], _, Context) -> Context;
 check_numbers_uniqueness([Number|Numbers], JObjs, Context) ->
+    io:format("MARKER:cb_callflows.erl:445 ~p~n", [Number]),
     case lists:dropwhile(fun(J) -> is_number_unique(J, Number) end
                          ,JObjs
                         )
     of
         [] -> check_numbers_uniqueness(Numbers, JObjs, Context);
         [JObj|_] ->
+            io:format("MARKER:cb_callflows.erl:451 ~p~n", [JObj]),
```
Comment (fenollp) [inline applications/crossbar/src/modules/cb_callflows.erl]: rm these

File: core/kazoo_documents-1.0.0/src/kzd_callflow.erl (hunk 1 above)
Comment (macpie) [inline core/kazoo_documents-1.0.0/src/kzd_callflow.erl]: Can't do that but I can add a default 'false' value

**Comment by fenollp** [comment]
@macpie Can you add a comment in the code where you can't do get_is_true ?

**Comment by macpie** [comment]
@fenollp Pb is that a wh_json:is_true on a json object does not return true

**Comment by fenollp** [comment]
@macpie No I get it, it's just that it's not something I get from just the code, so a comment'd be nice.

**Pull request merged**